run_sdk_container

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -eu
+
+cd $(dirname "$0")
+source sdk_lib/sdk_container_common.sh
+
+version="$(get_version_from_versionfile)"
+arch="all"
+checkout="true"
+create_only="false"
+name=""
+
+usage() {
+    echo "  $0 - Start an SDK container of a given SDK release version."
+    echo "       This will create the container if it does not exist, otherwise it will just start it."
+    echo "       If the container is already running then it will exec into the container."
+    echo "  Usage:"
+    echo "  $0 [-v <version>] [-a arch] [-n <name> ] [-c] [-C] [container-command]"
+    echo
+    echo "      container-command - command to be run in the container instead of"
+    echo "                     an interactive shell."
+    echo "      -v <version> - SDK version to use. Defaults to '$version'"
+    echo "                     (from '$sdk_container_common_versionfile')"
+    echo "      -a <amd64|arm64|all> - Target architecture (board support) of the SDK."
+    echo "                     'all' (the default) contains support for both amd64 and arm64."
+    echo "      -n <name>    - Custom name to use for the container."
+    echo "      -c           - Use scripts and ebuild repos as-is, do not check out version tag."
+    echo "      -C           - Create the container if it does not exist, then quit."
+    echo "                     (do not attach, run commands, etc)"
+    echo
+}
+# --
+
+while [ 0 -lt $# ] ; do
+    case "$1" in
+    -h) usage; exit 0;;
+    -v) version="$2"; shift; shift;;
+    -a) arch="$2";    shift; shift;;
+    -n) name="$2";    shift; shift;;
+    -c) checkout="false";    shift;;
+    -C) create_only="true";  shift;;
+    *) break;;
+    esac
+done
+
+if [ -z "$name" ] ; then
+    name="flatcar-sdk-${arch}-${version}"
+fi
+
+lock_version "$version" "$name"
+
+create_versionfile "$version"
+
+if $checkout; then
+    checkout_ebuild_repos "$version"
+fi
+
+stat="$(docker ps --all --no-trunc --filter name="$name" --format '{{.Status}}'\
+        | cut -f1 -d' ')"
+
+if [ -z "$stat" ] ; then
+    yell "Creating a new container '$name'"
+    mkdir -p "__build__/images"
+    docker create -ti \
+       -v /dev:/dev \
+       -v "$(pwd)/sdk_container:/mnt/host/source/" \
+       -v "$(pwd)/__build__/images:/mnt/host/source/src/build" \
+       -v "$(pwd):/mnt/host/source/src/scripts" \
+       --privileged \
+       --network host \
+       -e SDK_USER_ID="$(id -u)" \
+       -e SDK_GROUP_ID="$(id -g)" \
+       --name="$name" \
+       --hostname "flatcar-sdk-${arch}-${version}" \
+       "$sdk_container_common_registry"/flatcar-sdk-"${arch}:${version}"
```

Comment:
We need a way to forward a list of environment variables, both here and in `/etc/sudoers.d/90_env_keep` (see https://github.com/kinvolk/mantle/blob/flatcar-master/sdk/create.go#L72)

Reply:
There is also some mounting for ssh agent and gpg done: https://github.com/kinvolk/mantle/blob/flatcar-master/sdk/enter.go#L190

Reply:
`CopyGoogleCreds` is probably also relevant to upload binary packages

Reply:
Most of these can be passed via the `sdk_container`  volume mount (which ends up at `/mnt/host/source` in the container). Environment variables are trickier since these need to survive `sudo su ...` in the entrypoint. I'll come up with something.

Reply:
Added creds.